sdk/src/Member.cs

```diff
@@ -1592,201 +1337,162 @@ public BankInfo GetBankInfoBlocking(string bankId)
-        /// <param name="state">the state</param>
-        /// <returns>the signature</returns>
-        public Task<Signature> SignTokenRequestState(
-            string tokenRequestId,
-            string tokenId,
-            string state)
+        /// <param name="accountId">the account id</param>
+        /// <returns>a list of transfer endpoints</returns>
+        public IList<TransferEndpoint> ResolveTransferDestinationsBlocking(string accountId)
         {
-            return client.SignTokenRequestState(tokenRequestId, tokenId, state);
+            return ResolveTransferDestinations(accountId).Result;
         }
         
         /// <summary>
-        /// Signs a token request state payload.
+        /// Deletes the member
         /// </summary>
-        /// <param name="tokenRequestId">the token request id</param>
-        /// <param name="tokenId">the token id</param>
-        /// <param name="state">the state</param>
-        /// <returns>the signature</returns>
-        public Signature SignTokenRequestStateBlocking(
-            string tokenRequestId,
-            string tokenId,
-            string state)
+        /// <returns>Task</returns>
+        public Task DeleteMember()
         {
-            return SignTokenRequestState(tokenRequestId, tokenId, state).Result;
+            return client.DeleteMember();
         }
 
         /// <summary>
-        /// Gets all paired devices.
+        /// Deletes the member
         /// </summary>
-        /// <returns>a list of devices</returns>
-        public Task<IList<Device>> GetPairedDevices()
+        /// <returns>Task</returns>
+        public void DeleteMemberBlocking()
         {
-            return client.GetPairedDevices();
-        }
-        
-        /// <summary>
-        /// Gets all paired devices.
-        /// </summary>
-        /// <returns>a list of devices</returns>
-        public IList<Device> GetPairedDevicesBlocking()
-        {
-            return GetPairedDevices().Result;
+            DeleteMember().Wait();
         }
 
         /// <summary>
-        /// Verifies an affiliated TPP.
+        /// Creates a test bank account in a fake bank and links the account.
         /// </summary>
-        /// <param name="memberId">member ID of the TPP verify</param>
-        /// <returns>a task</returns>
-        public Task VerifyAffiliate(string memberId)
-        {
-            return client.VerifyAffiliate(memberId);
-        }
-        
-        /// <summary>
-        /// Verifies an affiliated TPP.
-        /// </summary>
-        /// <param name="memberId">member ID of the TPP verify</param>
-        /// <returns>a task</returns>
-        public void VerifyAffiliateBlocking(string memberId)
+        /// <param name="balance"></param>
+        /// <param name="currency"></param>
+        /// <returns>the linked account</returns>
+        public Task<Account> CreateTestBankAccount(string balance, string currency)
```

Comment:
Discussed with @gianluca-token offline, let's remove this method too.

Reply:
Just remove it from the `Member`, or should i remove it from `Client.cs` as well?

Reply:
yes, this is not used by anyone.